modules/tracker/Notification.php

```diff
@@ -25,6 +25,7 @@
  *
  * @property Tracker_ScalarModel $Tracker_Scalar
  * @property Tracker_TrendModel $Tracker_Trend
+ * @property Tracker_SubmissionModel $Tracker_Submission
  */
```

Comment:
I didn't notice this before when I added the moduleModels.  Can you add

```
* @property Tracker_AggregateMetricSpecModel $Tracker_AggregateMetricSpec
* @property Tracker_AggregateMetricModel $Tracker_AggregateMetric
```

?